zkstats/ops.py

```diff
@@ -25,24 +26,35 @@ def ezkl(self, x: list[torch.Tensor]) -> IsResultPrecise:
 
 class Where(Operation):
     @classmethod
-    def create(cls, x: list[torch.Tensor], error: float) -> 'Where':
+    def create(cls, x: list[torch.Tensor], error: float, precal_witness:dict = None, op_dict:dict = {}) -> 'Where':
         # here error is trivial, but here to conform to other functions
-        return cls(torch.where(x[0],x[1], MagicNumber ),error)
+        # just dummy result, since not using it anyway because we dont want to expose direct result from where
+        return cls(torch.tensor(1),error)
     def ezkl(self, x:list[torch.Tensor]) -> IsResultPrecise:
-        bool_array = torch.logical_or(x[1]==self.result, torch.logical_and(torch.logical_not(x[0]), self.result==MagicNumber))
-        # print('sellll: ', self.result)
-        return torch.sum(bool_array.float())==x[1].size()[1]
+        return torch.tensor(True)
+
 
 
 class Mean(Operation):
     @classmethod
-    def create(cls, x: list[torch.Tensor], error: float) -> 'Mean':
+    def create(cls, x: list[torch.Tensor], error: float, precal_witness:dict = None, op_dict:dict = {} ) -> 'Mean':
```

Comment:
Should use `None` in args and initiate the empty dict in the code if it's None. Otherwise all later calls to `Mean.create` will get the same dict from the first call.
```suggestion
    def create(cls, x: list[torch.Tensor], error: float, precal_witness: dict = None, op_dict: dict = None) -> 'Mean':
        if op_dict is None:
            op_dict = {}
```

Reply:
I agree that we should use None for op_dict default since we're not editing anything in ops.py, but we shouldnt initialize op_dict in ops.py. will change accordingly